engine/battle/battle_transition.asm

```diff
@@ -21,7 +21,7 @@ DoBattleTransition:
 	ld hl, hVBlank
 	ld a, [hl]
 	push af
-	vc_hook FPA_link_fight_begin
+	vc_hook Reduce_battle_transition_flashing
```

Comment:
Let's standardize on either `Sentence_case` or `lower_case` hook names (any preference for which?).

Reply:
I think I like `Sentence_case`